store/zksync/tokens.ts

```diff
@@ -30,7 +30,20 @@ export const useZkSyncTokensStore = defineStore("zkSyncTokens", () => {
         $fetch(`${eraNetwork.value.blockExplorerApi}/tokens?minLiquidity=0&limit=100&page=2`),
         $fetch(`${eraNetwork.value.blockExplorerApi}/tokens?minLiquidity=0&limit=100&page=3`),
       ]);
-      explorerTokens = responses.map((response) => response.items.map(mapApiToken)).flat();
+      const baseTokenData = await $fetch(
```

Comment:
Why is it needed? It should be included in the tokens list response

Reply:
For both gateways ZK is present in the tokens API response, so I think we can remove this logic